Sources/Private/Experimental/Animations/TransformAnimations.swift

```diff
@@ -151,17 +151,41 @@ extension CALayer {
         // Lottie animation files express scale as a numerical percentage value
         // (e.g. 50%, 100%, 200%) so we divide by 100 to get the decimal values
         // expected by Core Animation (e.g. 0.5, 1.0, 2.0).
+        //  - For some reason, negative `scaleX` values aren't applied by
+        //    Core Animation. Instead, we set up a `rotationY` animation below
+        //    to flip the view horizontally, which gives us the desired effect.
         CGFloat(scale.x) / 100
       },
       context: context)
 
+    // When `scale.x` is negative, we have to rotate the view
+    // half way around the y axis to flip it horizontally.
+    //  - We don't do this in snapshot tests because it breaks the tests
+    //    in surprising ways that don't happen at runtime. Definitely not ideal.
+    if !SnapshotTests.snapshotTestsAreRunning {
```

Comment:
This commit shows what happens when you run the snapshot tests with this behavior enabled: https://github.com/airbnb/lottie-ios/pull/1504/commits/a4a598ab0cdbbf002efcbd2d569ed19c356682d9

This works correctly at runtime in the Example app so it's unclear why this wouldn't work in the snapshot tests. I think the tests appear more broken with this code enabled than with it disabled.

Reply:
Maybe we could print something in the `else` to the console indicating that this feature isn't going to function correctly in the snapshot tests (not respecting negative scale.x)? May help prevent someone from having to dig through the rendering implementation to figure out what's going on